plumbing/object.go

```diff
@@ -12,6 +12,37 @@ var (
 	ErrInvalidType = errors.New("invalid object type")
 )
 
+// ObjectToPack is a representation of an object that is going to be into a
+// pack file. If it is a delta, Source is the delta source and Original is the
+// delta target
+type ObjectToPack struct {
```

Comment:
I really don't understand the goal of this struct, is this struct useful for non-delta objects?